components/src/dynamo/vllm/handlers.py

```
@@ -825,6 +825,382 @@ async def stop_profile(self, body: dict) -> dict:
+                await self.engine_client.remove_lora(lora_id)
+                del self.loaded_loras[lora_name]
+
+                # Unregister the MDC published on load so the frontend stops
+                # routing `model=<lora_name>` requests to this worker.
+                if self.generate_endpoint is not None:
+                    try:
+                        await unregister_model(
+                            endpoint=self.generate_endpoint,
+                            lora_name=lora_name,
+                        )
+                    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)